addons/web/static/src/views/fields/many2many_tags/many2many_tags_field.js

```diff
@@ -159,6 +159,7 @@ export class Many2ManyTagsField extends Component {
                 }
                 this.onTagKeydown(ev);
             },
+            hoverText: record.data.tooltip || record.data.display_name,
```

Comment:
I'm not sure about this whole PR to be honest. It fixes a pseudo issue by introducing a new feature. I don't know why they introduced that field in documents, as the widget never supported this (afaik).

But anyway, this line is wrong. You hardcode the field "tooltip" instead of accessing the field name given in options.